test/package/test_save_load.py

```diff
@@ -68,6 +68,19 @@ def test_save_module(self):
         self.assertEqual(package_a_i.result, "package_a")
         self.assertIsNot(package_a_i, package_a)
 
+    @skipIf(IS_FBCODE or IS_SANDCASTLE, "Tests that use temporary files are disabled in fbcode")
+    def test_dunder_imports(self):
+        filename = self.temp()
+        with PackageExporter(filename, verbose=True) as he:
```

Comment:
We should clean up the body of this test: 
- verbose=False
- no print
- We should have assertions that the `__imported__`'d modules are accessible from the resulting package.

Reply:
For a module to be accessible, do you mean that somethiing along the lines of `hi.import_module("subpackage_2") should work?